lib/utils/getGenerators.js

```diff
@@ -16,13 +16,13 @@ async function getGenerators() {
       if (name) {
         packages.push({ name, version: normalizeVersion(version) })
       }
-    })),
-    ...(getFiles(repoPath).map(async folder => {
-      const { name, version } = (await getPkg(folder))
+    }),
+    ...getFiles(repoPath).map(async folder => {
+      const { name, version } = await getPkg(folder)
       if (name) {
         repos.push({ name, version: normalizeVersion(version) })
       }
-    }))
+    })
   ])
 
   return { packages, repos }
```

Comment:
the generaors info are actually stored in `/Users/$USERNAME/Library/Preferences/sao-nodejs/config.json` by `lib/store.js`, maybe we could use that instead since it includes local generators.

sample:

```json
{ 
  "generators": {
    "096eb1e0": {
      "type": "npm",
      "name": "sao-nm",
      "version": "",
      "slug": "sao-nm",
      "subGenerator": false,
      "hash": "096eb1e0",
      "path": "/Users/egoist/.sao/V1/packages/096eb1e0/node_modules/sao-nm"
    },
    "7239d73c": {
      "type": "npm",
      "name": "sao-generator",
      "version": "",
      "slug": "sao-generator",
      "subGenerator": false,
      "hash": "7239d73c",
      "path": "/Users/egoist/.sao/V1/packages/7239d73c/node_modules/sao-generator"
    },
    "a3448d32": {
      "type": "local",
      "path": "/Users/egoist/dev/sao-react",
      "hash": "a3448d32"
    },
    "78c8ed0e": {
      "type": "repo",
      "slug": "nmrshll/frontend-scaffolds#react-tailwind",
      "hash": "78c8ed0e",
      "path": "/Users/egoist/.sao/V1/repos/78c8ed0e"
    }
  }
}
```

Reply:
Nice!